model/registry/gen-ai.yaml

```diff
@@ -85,3 +105,9 @@ groups:
         note: It's RECOMMENDED to format completions as JSON string matching [OpenAI messages format](https://platform.openai.com/docs/guides/text-generation)
         examples: ["[{'role': 'assistant', 'content': 'The capital of France is Paris.'}]"]
         tag: llm-generic-events
+      - id: operation.name
```

Comment:
we should reference it on spans as well, but it'd be ok with me to follow up in a different PR